tools/downloader/README.md

```diff
@@ -424,6 +424,9 @@ describing a single model. Each such object has the following keys:
 
 * `name`: the identifier of the model, as accepted by the `--name` option.
 
+* `composite_model_name`: the identifier of the composite model name, if the model is a composition
```

Comment:
This explanation is mixing the concepts. The _composite model_ is the composition of several models; the _model_ is a component of that composition.

Reply:
Fixed